Docs/docs.css

```diff
@@ -0,0 +1,194 @@
+    /* To override the padding given to the `tr-header` class */
+    padding: var(--space-sm) var(--space-lg);
+    
+}
+
+/* Main Header */
+.main-header{
+    box-shadow: var(--bs-light);
+    border-bottom-left-radius: 50% 20%;
+    border-bottom-right-radius: 50% 20%;
+    /* Fallback value for padding */
+    padding: var(--space-sm) var(--space-lg); 
+    padding: var(--space-sm) clamp(var(--space-lg), -0.875rem + 8.33vw, var(--space-xxlg));
+}
+
+/* Designing the logo  */
+.header-brand{
+    font-family: "Montserrat Alternates", sans-serif;
+    display: flex;
+    gap: 0.3rem;
+    align-items: center;
+    position: relative;
+    right: 1.25rem;
+}
+
+.logo{
+    width: 3.5rem;
+    position: relative;
+    left: 1.25rem;
+    bottom: 0.3rem;
+}
+
+.header-brand span{
+    position: relative;
+    z-index: 1;
+}
+.header-brand span:nth-of-type(2){
+    color: var(--tr-accent-color);
+}
+
+/*******************Main content**********************/
+
+.main-content{
+    padding: clamp(var(--space-lg), -0.875rem + 8.33vw, var(--space-xxlg));
+    gap: var(--space-xlg);
+    flex-wrap: wrap;
+    justify-content: center;
+}
+
+
+/* Buttons to change the preview variant  */
+.toggle-variant-btn{
+    background-color: transparent;
+    color: var(--tr-primary-color);
+    border: solid 1px var(--tr-primary-color);
+}
+
+.toggle-variant-btn:hover{
+    background-color: var(--tr-primary-overlay);
+}
+
+.toggle-variant-btn.active{
+    background-color: var(--tr-primary-color);
+    color: var(--tr-white);
+}
+
+/* Container containing preview and code blocks for component  */
+.main-demo-content{
+    flex: 1 1 70%;
+    flex-direction: column;
+    gap: var(--space-sm);
+    max-width:100%;
+    overflow: auto;
+    align-items: center;
+}
+
+.tabs-btn-list{
+    box-shadow: var(--bs-light);
+    width: 100%;
+}
+
+.tabs-btn-list .tr-btn{
+    border-radius: 0;
+    flex:1;
+}
+
+.tabs-btn-list .tr-btn:hover{
+    background-color: hsla(180, 65%, 48%, 0.2);
+}
+.tabs-btn-list .tr-btn:focus{
+    background-color: var(--tr-accent-overlay);
+    box-shadow: none;
+}
+
+.tabs-btn-list .tr-btn.active{
+    border-color: transparent;
+    background-color: hsla(180, 65%, 48%, 0.2);
+}
+
+.bottom-border{
+    background-color: var(--tr-accent-color);
+    bottom: 0;
+    height: 3px;
+    left: 0;
+    position: absolute;
+    transition: all ease 0.3s;
+    width: 50%;
+}
+
+.tab{
+    flex-grow: 1;
+    justify-content: center;
+    align-items: center;
+    overflow: auto;
+    max-height: 70vh;
+    padding: var(--space-xlg);
+    max-width:100%;
+    position: relative;
+}
+
+.code-blocks{
+    justify-content: start;
+    align-items: start;
+}
+
+.hide{
+    display: none;
+}
+
+.tabs-btn-list .tr-icon-label-btn{
+    justify-content: center;
+}
+
+.copy-btn{
+    position: absolute;
+    right: 30px;
+    top: 30px;
+    font-size: 2rem;
+    color: var(--tr-white);
+}
+
+.copy-btn:hover{
+    color: #008080;
```

Comment:
You can use variable for this color rather than using it directly

Reply:
fixed in the latest commit